modules.d/90mdraid/mdraid_start.sh

```diff
@@ -65,6 +70,14 @@ _md_force_run() {
             _md_start "${_md}"
         done
     fi
+
+    # try to assemble all the left md devices that should be assembled 
+    # probably failed because of timing issue in the first beginning
+    if [ -n "$_LEFT_UUID" ]; then
+        for _uuid in ${_LEFT_UUID}; do
```

Comment:
Add `local _uuid` to the beginning of the function, where the other local variables are defined.